### ISSUE TITLE
[bug] Unable to install datasets

### ISSUE THREAD

**Issue by abhi18av** [bactopia/bactopia#217]
Hi Bactopia team,

First off thanks for this wonderful tool! I have been really eager to try this out 😊 

However, when I tried to setup this on a server, I ran upon a glitch. I'd like to know whether there's anything I could change to get the databases setup locally.

**Description**

The installation process for datasets fails.

**Steps to Reproduce**

```
(bactopia) root@e2e-79-152:~/projects# bactopia datasets
2021-05-14 00:47:24:root:INFO - Setting up Ariba datasets
2021-05-14 00:47:24:root:INFO - Setting up vfdb_core Ariba dataset
2021-05-14 00:52:10:root:INFO - Setting up card Ariba dataset
2021-05-14 00:53:02:root:INFO - Setting up pre-computed Genbank/Refseq minmer datasets
2021-05-14 00:54:31:root:INFO - Setting up pre-computed PLSDB (plasmids) datasets
External command failed with exit code 1!

Command:
bash -c 'rm plsdb.zip'

Standard error:
rm: cannot remove 'plsdb.zip': No such file or directory
```

**Expected Behavior**

Complete the installation.

**Execution Environment**

- Bactopia Version: `1.7.0`
- OS: `Ubuntu - 18.04`
- Environment: `conda`

**Comment by abhi18av** [comment]
Update: For the time being I'm making progress by using the `bactopia datasets --skip_plsdb ` option.

**Comment by rpetit3** [comment]
Hi @abhi18av !

This is probably due to removing PLSDB from the main pipeline in v1.7.0. Thank you very much for bringing to my attention. I'll get a fix for it soon.

Robert

**Comment by abhi18av** [comment]
Oh, I see. Happy to see that this is nothing serious :)

**Comment by rpetit3** [comment]
Hi @abhi18av 

This should be fixed in v1.7.1 (https://github.com/bactopia/bactopia/releases/tag/v1.7.1)

Please feel free to reopen if not.

Thank you! 
Robert

**Comment by abhi18av** [comment]
Thanks for the resolution @rpetit3 ! 🎉
